src/pages/data-visualization/chart-types/index.mdx

```diff
@@ -130,14 +140,27 @@ Start by identifying the purpose of the visualization and choose a chart type ap
     href="/data-visualization/basic-charts#pie"
     >
 
-![Pie](images/donut.svg)
+![](images/pie.svg)
 
 </OverviewCard>
 </Column>
+
 <Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
  <OverviewCard
+    title="Stacked bar (horizontal)"
+    href="/data-visualization/basic-charts/#stacked-bar"
+    >
+
+![](images/stacked-bar-horizontal.svg)
```

Comment:
```suggestion
![Stacked bar (horizontal)](images/stacked-bar-horizontal.svg)
```

Reply:
also is this pointing to the horizontal demo? the link doesn't look like a horizontal demo link